README.md

```diff
@@ -116,6 +114,28 @@ Default: `null`
 
 Type: `str`
 
+### `mssql_firewall_configure`
+
+Whether to open the `mssql_tcp_port` port in the Linux firewall.
+
+When this variable is set to `true`, the role enables firewall even if it was not enabled.
+
+The role uses the System Roles firewall role to manage the firewall, hence, only firewall implementations supported by the firewall role work.
+
+If you set this variable to `false`, you must open the port defined with the `mssql_tcp_port` variable prior to running this role.
+
+Default: `false`
+
+Type: `bool`
+
+#### `mssql_firewall_close_previous`
+
+Whether to close the firewall port that was previously configured in SQL Server in case you change from one port to another when providing the `mssql_firewall_configure` variable.
+
+Default: `false`
+
+Type: `bool`
```

Comment:
I introduced this for users who need this port opened, e.g. if they use it for something else. I don't think people generally use the same port for multiple services, but it can be possible.
Also, this way, user accepts that some port will be closed. 
What do you think?

Reply:
Okay I agree, adding it was an overkill. Removed `mssql_firewall_close_previous`.